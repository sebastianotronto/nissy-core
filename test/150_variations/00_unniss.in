unniss
Uw2 F B' L x M R D' (D' R2 F' L2 B D')
