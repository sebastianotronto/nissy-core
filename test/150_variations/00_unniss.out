1
Uw2 F B' L x M R D' D B' L2 F R2 D
